src/snowflake/cli/_plugins/nativeapp/project_model.py

```diff
@@ -129,12 +134,13 @@ def project_identifier(self) -> str:
         # sometimes strip out double quotes, so we try to get them back here.
         return to_identifier(self.definition.name)
 
-    @cached_property
+    @property
```

Comment:
removed caching to avoid caching the suffix in tests, since it will contain a unique ID for each test